.github/workflows/doc-publish.yml

```diff
@@ -134,4 +135,3 @@ jobs:
       - name: Report deployment url
         run: |
           echo "::notice::The live documentation has been deployed - ${{ steps.deploy-netlify.outputs.NETLIFY_URL }}"
```

Comment:
Can we also copy [this](https://github.com/sagemath/sage/pull/37387/files#diff-7282d89989d4658ef93d9d1933ae0079effcb523f9723c84130e0c1508d94992R84) over so that it updates the documentation comment with the link to the "changes" page?